README.md

```diff
@@ -1558,6 +1665,61 @@ An Atomic Long usage example is shown below.
     std::cout << "Count is:" << counter.get() << std::endl; // Counter is 1000000
 ```
 
+#### 7.4.10.1. Using Atomic Long Non-Blocking Async Methods
+IAtomicLong provides asynchronous methods where you can execute a bunch of non-blocking methods and get the response later in your code or just let the atomic long operations complete at the background. You can also use callbacks to be executed upon the completion of the atomic long operations.
```

Comment:
Hazelcast IAtomicLong provides ... _(for the sake of consistency with the other newly added sections' intros)._